test/auth/auth.spec.js

```diff
@@ -0,0 +1,116 @@
+/* eslint-disable no-undef */
+let chai = require('chai')
+
+let chaiHttp = require('chai-http')
+
+// eslint-disable-next-line no-unused-vars
+let server = require('../../src/server.js')
+
+const { editUser } = require('../../src/controllers/user/handlers')
+
+let should = chai.should()
+
+chai.use(chaiHttp)
+
+var data = require('./shared.js')
+
+const application = require('../../src/app')
+
+const { PORT } = process.env
+let app = 'http://localhost:' + PORT
+
+let basePath = '/v1/auth'
+
+describe('Testing auth', () => {
+  describe('test auth/register-local', () => {
+    /**
+     * Test creating new user
+     */
+    describe('Test creating new user (customer) ', () => {
+      it('Creating new user (customer) ', (done) => {
```

Comment:
Best practise for testing `it` is to write like this:

`it('should create a customer')`

You should make research about it. I found quickly [this one](https://yeoman.io/contributing/testing-guidelines.html) but there are probably more relevant.